include/pooler.h

```diff
@@ -22,6 +22,7 @@ void resume_pooler(void);
 void suspend_pooler(void);
 void per_loop_pooler_maint(void);
 void pooler_tune_accept(bool on);
+void cleanup_tcp_sockets(void);
```

Comment:
It seems unrelated to this change.

Reply:
Thanks for noticing, that one should be part of #902 only.